README.md

```diff
@@ -162,6 +163,36 @@ AWS_ACCESS_KEY_ID=your-access-key
 AWS_SECRET_ACCESS_KEY=your-secret-access-key
 ```
 
+### Databricks
+
+CARTO Analytics Toolbox for Databricks provides geospatial functionality leveraging the GeoMesa SparkSQL capabilities. It implements Spatial Hive UDFs. In order to install the toolbox the library (jar-with-dependencies) needs to be installed in the cluster you are using, and the Hive UDFs registered via createUDFs sql script
+
+**Tools**
+
+Make sure you have installed the following tools:
+
+- `make`: https://www.gnu.org/software/make/
+- `jdk (8 or 11)`: https://www.oracle.com/java/technologies/javase/javase8u211-later-archive-downloads.html (v8.x)
+- `sbt`: https://www.scala-sbt.org/1.x/docs/Setup.html (v1.x)
+- `Python2.7.9 and above`: https://www.python.org/downloads/release/python-2718/ (v2.7.18)
```

Comment:
Is python 2 really necessary?

Reply:
You need either python2 or python 3, but now I think that it is better to remove it and assume that everybody will install the databricks CLI with python 3. 
Thanks!